projects/micro-dash/src/lib/lang/to-string.spec.ts

```diff
@@ -0,0 +1,21 @@
+import { toString } from "./to-string";
+
+describe("toString()", () => {
+  it("works", () => {
+    expect(toString("already a string")).toBe("already a string");
+    expect(toString("")).toBe("");
+    expect(toString(1)).toBe("1");
+    expect(toString([1, 2, 3])).toBe("1,2,3");
+    expect(toString([undefined, null])).toBe(",");
+    expect(toString({ hi: "there" })).toBe("[object Object]");
+    expect(toString({ toString: () => "custom toString()" })).toBe(
+      "custom toString()",
+    );
+  });
+
```

Comment:
There is a standard comment to put here to cite the tests we copy from lodash. Check any other spec file for the verbiage.